public/javascripts/accessibility_error_summary.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * This script solves a problem
+ * the Select tax to pay page
+ *  error summary section
+ * The href has a value which does not
+ * match the id of the first radio button
+ */
+
+window.addEventListener("load", function(event) {
```

Comment:
This is a script that's specific to the `direct-debit-frontend` service for accessibility workarounds, so should be removed.